libr/anal/p/anal_riscv.c

```diff
@@ -58,9 +58,11 @@ static int riscv_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int le
 	op->addr = addr;
 	op->type = R_ANAL_OP_TYPE_UNK;
 
-	if (len>=sizeof (word))
-		r_mem_copyendian ((void*)&word, (const void*)data, sizeof (word), true);
-	else r_mem_copyendian ((void*)&word, (const void*)data, 2, true); //sizeof (word), true);
+	if (len >= sizeof (word)) {
+		word = anal->big_endian? read_be64(data): read_le64(data);
```

Comment:
maybe add a helper function instead of checking endian and using different calls